astronomer/providers/databricks/operators/databricks.py

```diff
@@ -12,6 +15,48 @@
 from astronomer.providers.utils.typing_compat import Context
 
 
+def _handle_non_successful_teminal_states(
```

Comment:
```suggestion
def _handle_non_successful_terminal_states(
```

Reply:
does it makes sense to move this took the hook, not sure but maybe if multiple operators need we can reuse it wdyt?